### PR TITLE
NOISSUE - Add Updated By Field

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Adds `updated_by` field for clients and groups

### Which issue(s) does this PR fix/relate to?
Resolves https://github.com/ultravioletrs/issues/issues/215

### List any changes that modify/break current functionality
Add `updated_by` field 

### Have you included tests for your changes?
Yes

### Did you document any new/modified functionality?
No

### Notes
N/A

### CODE REVIEW

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Please resolve conflicts so we ca merge.

**Comment by rodneyosodo** [comment]
CI is fixed by [`6e919ba` (#1781)](https://github.com/mainflux/mainflux/pull/1781/commits/6e919ba872434c328af544e22410687ea0a11490)

File: users/clients/postgres/clients.go [hunk 1]
```diff
@@ -30,13 +30,13 @@ func NewClientRepo(db postgres.Database) clients.ClientRepository {
 }
 
 func (repo clientRepo) Save(ctx context.Context, c clients.Client) (clients.Client, error) {
-	q := `INSERT INTO clients (id, name, tags, owner, identity, secret, metadata, created_at, updated_at, status, role)
-        VALUES (:id, :name, :tags, :owner, :identity, :secret, :metadata, :created_at, :updated_at, :status, :role)
-        RETURNING id, name, tags, identity, metadata, COALESCE(owner, '') AS owner, status, created_at, updated_at`
+	q := `INSERT INTO clients (id, name, tags, owner, identity, secret, metadata, created_at, updated_at, updated_by, status, role)
```
Comment (dborovcanin) [inline users/clients/postgres/clients.go]: No need for two cases. Just set the owner to nil if it's empty. Also, it makes sense to leave `updated_by` nullable and default to it on creation, indicating the Client has never been changed.

Comment (dborovcanin) [reply users/clients/postgres/clients.go]: The same goes for Things clients.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**